src/main/java/io/reactivex/rxjava3/core/Observable.java

```diff
@@ -10371,7 +10371,7 @@ public final Completable ignoreElements() {
     @CheckReturnValue
     @SchedulerSupport(SchedulerSupport.NONE)
     @NonNull
-    public final <R> Observable<R> map(@NonNull Function<? super T, ? extends R> mapper) {
+    public final <R> Observable<@NonNull R> map(@NonNull Function<? super T, ? extends R> mapper) {
```

Comment:
Thanks. I think it should be on the type argument declaration as <@NonNull R>. I have to check other cases and get back to you shortly.

Reply:
Ah yes you're right

```suggestion
    public final <@NonNull R> Observable<R> map(@NonNull Function<? super T, ? extends R> mapper) {
```